src/main/java/org/openjdk/jextract/impl/TreeMaker.java

```diff
@@ -261,8 +278,8 @@ final Type.Declared recordDeclaration(Cursor parent, Cursor recordCursor) {
                     if (fc.isAnonymousStruct()) {
                         // process struct recursively
                         pendingFields.add(recordDeclaration(parent, fc).tree());
-                    } else {
-                        Type fieldType = typeMaker.makeType(fc.type());
+                    } else if (!fc.isBitField() && !fc.spelling().isEmpty()) {
```

Comment:
Spurious change? Or was this condition needed after all?

Reply:
This (and other changes) have come back as a result of a bad merge I will fix